authors/apps/authentication/views.py

```diff
@@ -60,10 +60,20 @@ def post(self, request):
         # https://stackoverflow.com/questions/3005080/how-to-send-html-email-with-django-with-dynamic-content-in-it
         html_message = render_to_string(template_name, context)
         text_message = strip_tags(html_message)
-        mail.send_mail(
-            'Please verify your email',
-            text_message, settings.FROM_EMAIL,
-            [user_email, ], html_message=html_message)
+        # mail.send_mail(
+        #     'Please verify your email',
+        #     text_message, settings.FROM_EMAIL,
+        #     [user_email, ], html_message=html_message)
```

Comment:
@BrianSerem, kindly remove the commented out code.

Reply:
I have removed the commented piece of code. I have also made more changes.